spec/methods/auth/auth.ts

```diff
@@ -339,5 +342,200 @@ export namespace ConfirmPassReset {
    */
   export type Result = MessageResult;
 
+  export type Response = MessageResponse;
+}
+
+//////////////////////////////////
+// Configure MFA Types
+//////////////////////////////////
+
+/**
+ * Response from a call to begin
+ * App MFA setup.  It contains a secret code
+ * to enter into the MFA App.
+ */
+export type SecretCodeResult = {
```

Comment:
What do you think of putting this `Result`/`Response` pair with the other one at the top?  Could go either direction with that.